drivers/media/platform/bcm2835/bcm2835-unicam.c

```diff
@@ -687,18 +752,20 @@ static irqreturn_t unicam_isr(int irq, void *dev)
 {
 	struct unicam_device *unicam = (struct unicam_device *)dev;
 	struct unicam_cfg *cfg = &unicam->cfg;
-	struct unicam_dmaqueue *dma_q = &unicam->node[0].dma_queue;
 	unsigned int lines_done = unicam_get_lines_done(dev);
 	unsigned int sequence = unicam->sequence;
+	int num_nodes_streaming = unicam_num_nodes_streaming(dev);
+	u64 ts;
```

Comment:
I wasn't going to nit-pick over this one, but have had to raise issue below. Longest variable description to shortest, so ts should be after ista,sta.

Reply:
Done.